src/modules/auth/controllers/het-archief.controller.spec.ts

```diff
@@ -165,7 +175,11 @@ describe('HetArchiefController', () => {
 			mockUsersService.getUserByIdentityId.mockReturnValueOnce(archiefUser);
 			mockIdpService.determineUserGroup.mockReturnValueOnce(Group.CP_ADMIN);
 
-			const result = await hetArchiefController.loginCallback({}, samlResponse);
+			const result = await hetArchiefController.loginCallback(
+				{ path: '/auth/hetarchief', headers: {} } as unknown as Request,
```

Comment:
Avoid double cast, put this object into a separate const

Reply:
👍